geometry_msgs/Pose pose
int64 frame_num
